doc/designs/ddd/ddd-06-mem-uv.c

```diff
@@ -707,8 +708,8 @@ static void post_connect(APP_CONN *conn, int status, void *arg)
         return;
     }
 
-    wr = app_write(conn, tx_msg, sizeof(tx_msg)-1, post_write_get, NULL);
-    if (wr < sizeof(tx_msg)-1) {
+    wr = app_write(conn, tx_msg, mlen, post_write_get, NULL);
+    if (wr < mlen) {
```

Comment:
Likewise here...

Reply:
The variable `wr` was already present here (it is not something that I added).

If removing it makes things better, then I am happy to do that, but my preference is to leave it as is.

Maybe someone else can give an opinion here.